shared/chat/conversation/input/container.js.flow

```diff
@@ -5,7 +5,7 @@ export type OwnProps = {
   focusInputCounter: number,
   onEditLastMessage: () => void,
   onScrollDown: () => void,
-  fromManageChannels: ?boolean,
+  previousPath: ?(*[]),
```

Comment:
What syntax is this? Why isn't this a `Path` from `route-tree/index`

Reply:
heh, sorry, I was in the middle of this. Should have tagged WIP